crates/ilp-node/src/node.rs

```diff
@@ -200,6 +203,9 @@ pub struct InterledgerNode {
         alias = "redis_url"
     )]
     pub database_url: String,
+    /// Database prefix which can be used in case a db instance is shared by multiple nodes
+    #[serde(default = "default_database_prefix")]
```

Comment:
this could be simplified into:
```suggestion
    #[serde(default)]
```
which will call `<String as std::default::Default>::default()` with one less fn, if this was not yet considered.

Reply:
I agree about the parametrizing existing integration tests. I think it would make sense to do it in a separate PR, just to have more freedom if more good testing idea apears so it can be part of the same testing-related PR. Just 2 c

Reply:
Serde thingy is done :)